frontend/src/features/map_planning/layers/_frontend_only/grid/groups/Grid.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  RELATIVE_DOT_SIZE,
+  SEA_BLUE_500,
+} from '@/features/map_planning/layers/_frontend_only/grid/util/Constants';
+import { BoundsRect } from '@/features/map_planning/store/MapStoreTypes';
+import { ONE_METER, TEN_CENTIMETERS } from '@/features/map_planning/utils/Constants';
+import { Group, Line } from 'react-konva';
+
+export const Grid = (rect: BoundsRect) => {
+  let gridStep = TEN_CENTIMETERS;
+  if (rect.width > 100 * ONE_METER) {
+    gridStep = 10 * ONE_METER;
+  } else if (rect.width > 1000) {
+    gridStep = ONE_METER;
+  }
+
+  const gridDotSize = rect.width * RELATIVE_DOT_SIZE;
```

Comment:
At the very end of "1m" zoom (shortly before it jumps to 10m) the dots are too big.

Reply:
The dots are always the same size (unless the browser window size changes).
It just appears this way because they are too close together. 

I will make sure that the jump from one meter to ten meter scale happens a bit earlier.